client/scss/components/forms/_field-comment-control.scss

```diff
@@ -40,22 +37,45 @@ $button-size: theme('spacing.5');
     .tab-content--comments-enabled .w-field__input:focus-within > &,
     &:hover,
     &:focus,
-    &.w-field__comment-button--focused {
+    &.w-field__comment-button--focused,
+    .w-field--admin_model_chooser.w-field--commentable:hover & {
       visibility: visible;
       opacity: 1;
     }
   }
+
+  // Position comment icons in choosers beside the other actions
+  .tab-content--comments-enabled .chooser & {
+    position: unset;
+    inset-inline-end: 0;
+    margin-inline-start: 0;
+    transform: translateY(3px);
+  }
+
+  .tab-content--comments-enabled .unchosen:hover &,
+  .tab-content--comments-enabled .unchosen:focus-within &,
+  .tab-content--comments-enabled .chooser .chosen & {
+    opacity: 1;
+    visibility: visible;
+  }
```

Comment:
Can we rewrite as much of this as possible so it applies even without `.tab-content--comments-enabled `? In particular all the positioning feels to me like it doesn’t need that class.

Reply:
Actually none of these ones need `.tab-content--comments-enabled` you're right, i'll remove them